app/controllers/users_controller.rb

```diff
@@ -6,6 +6,7 @@ def new
 
   def create
     @user = User.new(user_params)
+    @user.regenerate_password_token
```

Comment:
This might fail in the unlikely event that the password token that's generated is the same as another user's.

Reply:
also, if this is mutating the `@user` instance in place, it's considered best practice to use a `!` (bang) method, like Ruby's `gsub!`. so calling this `regenerate_password_token!` here would be more inline with that.